wdio-ui5-service/src/lib/wdioUi5-index.js

```diff
@@ -519,6 +519,27 @@ function setup(context) {
         }
     });
 
+    if (context && !context.asControl) {
+        context.asControl = function (target) {
```

Comment:
Whats the point of not using `_context.addCommand('asControl` ... ?

Reply:
b/c we need a `sync` fn for entering into the `Proxy`-ied async call chain:
sync `asControl` → (Proxy) → async `_asControl` → async $UI5controlAPI.methods()